### ISSUE TITLE
Typo in Keras Lab Notebook

### ISSUE THREAD

**Issue by thomasantony** [udacity/sdc-issue-reports#162]
In the "Train the Network" section, the test for the code is:

assert(history.history['acc'][0] > 0.5), "The training accuracy was: %.3f" % history.history['acc']

It should be:

assert(history.history['acc'][0] > 0.5), "The training accuracy was: %.3f" % history.history['acc'][0]

The missing [0] at the end is causing errors even when the training code is correct.

**Comment by domluna** [comment]
Fixed in latest notebook.
